test/ApplicationTest.php

```diff
@@ -632,7 +625,7 @@ public function testHelpDisplaysInformationForCommandWithDependencies(): void
 
         $contains = [
             "Usage:\n  example:dep [options]\n",
-            "  -s, --string=STRING   A string option [default: \"default value\"]\n",
+            "  -s, --string=STRING        A string option [default: \"default value\"]\n",
```

Comment:
not sure why this change in test? Just different number of spaces?

Reply:
Due to the new `--container` parameter, the indentation is not correct anymore and the test fails.